docs/source/array-chunks.rst

```diff
@@ -268,12 +268,14 @@ are filled, we move across the "columns" first (along ``axis=1``), and then down
 to the next row.
 
 .. image:: images/reshape.png
+   :alt:
```

Comment:
```suggestion
   :alt: Visual representation of a 2-dimensional NumPy array being reshaped to 1 dimension. Arrows indicate the order in which values are from the original array are copied to the new array, moving across the columns in axis 1 first before moving down to the next row in axis 0. 
```

Reply:
Wondering if since these image contain code that they are meant to represent (in this case `np.ones(3, 4).reshape(12)`) if it would be good to include that code in the alt text? Or perhaps even better move the generating code to text outside of the image.